example/src/Vector.js

```diff
@@ -60,7 +60,7 @@ export class Vector extends Array {
 
 /**
  * A 3D vector
- * @typedef {Vector[]} VectorArray
+ * @typedef {Vector[]}VectorArray
```

Comment:
According to JSDoc:
```
@typedef [<type>] <namepath>
```
Here's whitespace, if this is related to #27, we have to fix issue without using temporary workarounds in another scopes.
As alternative add task to fix this part within #27

Reply:
I don't know what you said there 😅 but I'll fix it in another PR. It wasn't introduced with this PR.

Reply:
I meant this looks like a workaround for bug in #27 
Original line here is correct, so no need to make that workaround.

Reply:
This is indeed a workaround for #27. I just want the example docs to be generated correctly. I'll revert this once I fix #27.

Reply:
Got it.